test/test_torch.py

```diff
@@ -4499,12 +4499,6 @@ def test_masked_scatter(self, device, dtype):
                     # in a separate test
                     return
 
-                # TODO: update test when masked scatter is supported for complex
-                # and cpu supports half
-                if (dt == torch.half and self.device_type == 'cpu') or dt.is_complex:
```

Comment:
you probably still wanna retain this for `if (dt == torch.half and self.device_type == 'cpu')` right?

Reply:
oh nvm you also enable `masked_scatter` for `torch.half` on CPU

Reply:
Oh forgot to add it in the description but have also added support for half on CPU.

Reply:
Updated the description.